scripts/installer/set-disk-partitions

```diff
@@ -28,11 +28,11 @@ if [ "$PARTITION_COUNT" -gt "1" ]; then
             exit 1
         fi
     done
-
-    dd if=/dev/zero of=${DEVICE} bs=512 count=1
-    partprobe ${DEVICE}
 fi
 
+dd if=/dev/zero of=${DEVICE} bs=512 count=2048
```

Comment:
@ibuildthecloud Looks ok? 
If we're reformatting the target disk anyway, why not write a few zero bytes on it first to help GRUB out.